File: Upcoder/Lập trình cơ bản/3.Vòng lặp/Hình chữ nhật rỗng/NGOISAO3.py
# Nhập chiều dài và chiều rộng của hình chữ nhật
a, b = map(int, input().split())

for i in range(1, a + 1):
    for j in range(1, b + 1):
        if i == 1 or j == 1 or i == a or j == b:
            print("*", end="")
        else:
            print(" ", end="")
    if i < a:
        print()

File: Upcoder/Lập trình cơ bản/3.Vòng lặp/Hình chữ nhật rỗng/NGOISAO3.cs
using System;

namespace HCN
{
    public class Program
    {
        public static void Main(string[] args)
        {
            string []input = Console.ReadLine().Split(' ');
            int a = int.Parse(input[0]);
            int b = int.Parse(input[1]);

            for (int i = 1; i <= a; ++i)
            {
                for (int j = 1; j <= b; ++j)
                {
                    if (i == 1 || j == 1 || i == a || j == b)
                        Console.Write("*");
                    else
                        Console.Write(" ");
                }
                if (i < a)
                    Console.WriteLine();
            }
        }
    }
}
